README.md

```diff
@@ -71,6 +71,19 @@ let p = new Preprocessor();
 let output = p.parse('<template>Hi</template>');
 ```
 
+#### `PreprocessorOptions`
+
+````ts
+interface PreprocessorOptions {
+
+  /** Default is `false` */
+  inline_source_map: boolean;
```

Comment:
```suggestion
  inline_source_map?: boolean;
```

Reply:
ooo good catch, I didn't see this when I was reviewing 🙈